src/service/DefaultService.ts

```diff
@@ -178,102 +192,28 @@ export async function PackageRate(id: PackageID, xAuthorization: AuthenticationT
-  const examples: any = {};
-  examples['application/json'] = {
-    "metadata": {
-      "Version": "1.2.3",
-      "ID": "ID",
-      "Name": "Name",
-    },
-    "data": {
-      "Content": "Content",
-      "JSProgram": "JSProgram",
-      "URL": "URL",
-    },
-  };
-
   try{
-    let connection: any;
-    
-    // // Create a Promise to wrap the pool.getConnection call
-    // const getConnectionPromise = new Promise((resolve, reject) => {
-    //   pool.getConnection((err, conn) => {
-    //     if (err) {
-    //       reject(err);
-    //     } else {
-    //       connection = conn;
-    //       resolve(1);
-    //     }
-    //   });
-    // });
-
-    // // Wait for the connection to be established
-    // await getConnectionPromise;
-
-
-    interface test extends RowDataPacket {
-      ID: string;
-      Name: string;
-    }
 
-    // const [results, fields] = await promisePool.execute<ProcedureCallPacket<test[]>>(
-    //   'SELECT PM.*, PD.* FROM Package AS P JOIN PackageMetadata AS PM ON P.MetadataID = PM.ID JOIN PackageData AS PD ON P.DataID = PD.ID WHERE P.PackageID = ?',
-    //   [id]
-    // );
+    // interface test extends RowDataPacket {
+    //   ID: JSON;
+    // }
 
-    // const [results] = await promisePool.execute<ProcedureCallPacket<test[]>>('CALL GetPackage(?)', [
-    //   id,
-    // ]);
-
-    const isResultSetHeader = (data: unknown): data is ResultSetHeader => {
-      if (!data || typeof data !== 'object') return false;
-    
-      const keys = [
-        'fieldCount',
-        'affectedRows',
-        'insertId',
-        'info',
-        'serverStatus',
-        'warningStatus',
-        'changedRows',
-      ];
-    
-      return keys.every((key) => key in data);
-    };
-    
-    // results.forEach((users) => {
-    //   if (isResultSetHeader(users)) {
-    //     console.log('----------------');
-    //     console.log('Affected Rows:', users.affectedRows);
-    //   } else {  
-    //     users.forEach((user) => {
-    //       console.log('----------------');
-    //       console.log('id:  ', user.ID);
-    //       console.log('name:', user.Name);
-    //       console.log('URL: ', user.URL);
-    //     });
-    //   }
-    // });
-
-
-    // const [results, fields] = await promisePool.execute<test[]>('SELECT * FROM Package', []);
-
-    // const response = (results[0][0] as { response: YourResponseType }).response;
-    // console.log(typeof(results));
-    // console.log(results[0][0].v_JSON);
-    // const selectResult: RowDataPacket[] = results[0] as RowDataPacket[];
-
-  //   console.log(results);
-  //   // console.log(fields);
+    const [results] = await promisePool.execute<ProcedureCallPacket<RowDataPacket[]>>('CALL GetPackage(?)', [
+      id,
+    ]);
 
+    console.log(results)
 
-  //   return respondWithCode(200, results[0][0]);
+    if(results[0].length == 0) {
+      return respondWithCode(404);
+    }
+    else {
+      return respondWithCode(200, results[0][0]);
```

Comment:
Does this return in the JSON format specified in the swagger file? Something like below
{
  "metadata": {
    "Name": "Underscore",
    "Version": "1.0.0",
    "ID": "underscore"
  },
  "data": {
    "Content": 
    ...
    }

Reply:
Yes when looking at example.sql it will return this formatting